pkg/authz/warrant/service.go

```diff
@@ -92,26 +92,28 @@ func (svc WarrantService) Create(ctx context.Context, warrantSpec WarrantSpec) (
 			}
 		}
 
-		// Create referenced subject if it does not already exist
-		objectSpec, err := svc.ObjectSvc.GetByObjectTypeAndId(txCtx, warrantSpec.Subject.ObjectType, warrantSpec.Subject.ObjectId)
-		if err != nil {
-			var recordNotFoundError *service.RecordNotFoundError
-			if !errors.As(err, &recordNotFoundError) {
-				return err
-			}
-		}
-
-		if objectSpec == nil {
-			_, err = svc.ObjectSvc.Create(txCtx, object.CreateObjectSpec{
-				ObjectType: warrantSpec.Subject.ObjectType,
-				ObjectId:   warrantSpec.Subject.ObjectId,
-			})
+		// Unless subject objectId is wildcard, create referenced subject if it does not already exist
+		if warrantSpec.Subject.ObjectId != "*" {
```

Comment:
Can you replace `"*"` in this condition and in the one above (for creating the object) with `Wildcard`? It's a constant I added to the warrant spec file.